client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/index.jsx

```diff
@@ -190,9 +191,23 @@ class VisibleSubmissionEditIndex extends Component {
   onSaveDraft = (data) => {
     const {
       dispatch,
-      match: { params },
+      match: {
+        params: { submissionId },
+      },
+    } = this.props;
+
+    dispatch(saveDraft(submissionId, data));
+  };
+
+  onSaveAnswer = (data, answerId, currentTime) => {
```

Comment:
Can we define this in SubmissionAnswer directly? This handler is specific for the answer itself and need not to be defined on submission level